addons/mail/wizard/mail_followers_edit_views.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <!-- wizard view -->
+        <record model="ir.ui.view" id="mail_followers_edit_form">
+            <field name="name">Add/Remove Followers</field>
+            <field name="model">mail.followers.edit</field>
+            <field name="arch" type="xml">
+                <form string="Add/Remove Followers">
+                    <group>
+                        <field name="res_model" invisible="1"/>
+                        <field name="res_ids" invisible="1"/>
+                        <field name="operation" invisible="context.get('only_add_followers')" widget="radio" options="{'horizontal': true}"/>
```

Comment:
Would make a new view (inherited, primary) with higher priority (aka not chosen by default), with operation being invisible and button name updated. Then the action using it just has to use the right form view ID. This makes the view more standard, and we don't rely on another context key nobody knows. Instead we have views that are more explicit.